src/app/public/modules/lookup/types/lookup-show-more-context.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  SkyLookupSelectMode
+} from './lookup-select-mode';
+
+import {
+  SkyLookupShowMoreDefaultPickerConfig
+} from './lookup-show-more-default-picker-config';
+
+/**
+ * @internal
+ * Context for the show more modal. These values are provided by the lookup component.
+ */
+ export class SkyLookupShowMoreContext {
```

Comment:
Let's call this `SkyLookupShowMorePickerContext` just to be more specific.

Reply:
I actually went with `SkyLookupShowMoreNativePickerContext` to distinguish that this is only used for the native picker as we already had a `SkyLookupShowMoreCustomPickerContext`